test/compiled_shaders_include/kompute_test/shaders/shadertest_op_custom_shader.hpp

```diff
@@ -23,101 +23,103 @@
 
 namespace kp {
 namespace shader_data {
-static const unsigned char test_shaders_glsl_test_op_custom_shader_comp_spv[] = {
```

Comment:
Also automatically generated so we shoudl skip (all the compiled shared folders)